src/main/java/com/monumental/services/UserService.java

```diff
@@ -207,6 +202,63 @@ public VerificationToken verifyToken(String token) throws ResourceNotFoundExcept
         return verificationToken;
     }
 
+    /**
+     * Require that the current user has a specific Role, otherwise throw AccessDeniedException
+     * @param role - The Role to check for
+     * @throws AccessDeniedException - If the running User does not have the specified Role
+     * @throws UnauthorizedException - If there is no User in the session, i.e. logged out
+     */
+    public void requireUserIsInRole(Role role) throws AccessDeniedException, UnauthorizedException {
+        UserDetails session = this.getSession();
+        if (!session.getAuthorities().contains(null)) {
+            throw new AccessDeniedException("Access is denied");
+        }
+    }
```

Comment:
What's the point of taking in the `role` param here?

Reply:
The implementation here was temporary, I forgot to go back and put the real one in. Fixed now.